ntoskrnl/ke/amd64/trap.S

```diff
@@ -590,7 +590,7 @@ ENDFUNC
 PUBLIC KiApcInterrupt
 .PROC KiApcInterrupt
    /* No error code */
-    EnterTrap (TF_VOLATILES or TF_IRQL)
```

Comment:
At this point, there's no user left of TF_VOLATILES and TF_SAVE_ALL is used everywhere. Maybe we can just remove those two altogether and make `EnterTrap` save all registers unconditionally.

Reply:
True, I will review this later.